tox.ini

```diff
@@ -49,7 +50,7 @@ commands =
 [testenv:mypy]
 basepython = python3.11
 deps =
-    mypy==0.971
+    mypy==1.4.1
```

Comment:
This is also frozen in line 11, where upgrading causes issues with `tox -e py` on 3.11 for me.